pkg/manager/manager.go

```diff
@@ -44,6 +44,10 @@ const (
 	OSContainerAnnotationKey = "runtime.frakti.alpha.kubernetes.io/OSContainer"
 	// The annotation value specifying this pod will run by OS container runtime.
 	OSContainerAnnotationTrue = "true"
+	// The annotation key specifying this pod will run by unikernel runtime.
+	UnikernelAnnotationKey = "runtime.frakti.alpha.kubernetes.io/Unikernel"
+	// The annotation value specifying this pod will run by unikernel runtime.
+	UnikernelAnnotationTrue = "true"
```

Comment:
Why do we need this? we should decide the unikernel runtime based on image.

Reply:
@feiskyer In RunPodSandbox, there is no container image yet, so we need to have some flag to indicate which runtime we are using.

Reply:
hmm, good catch